assemble/src/main/assemblies/component.xml

```diff
@@ -40,15 +40,48 @@
         <include>org.apache.commons:commons-math3</include>
         <include>org.apache.commons:commons-vfs2</include>
         <include>org.apache.thrift:libthrift</include>
+        <include>org.eclipse.jetty:jetty-continuation</include>
         <include>org.eclipse.jetty:jetty-http</include>
         <include>org.eclipse.jetty:jetty-io</include>
         <include>org.eclipse.jetty:jetty-security</include>
         <include>org.eclipse.jetty:jetty-server</include>
+        <!-- jetty-servlet only needed by old monitor -->
         <include>org.eclipse.jetty:jetty-servlet</include>
         <include>org.eclipse.jetty:jetty-util</include>
         <include>org.apache.htrace:htrace-core</include>
         <include>org.slf4j:slf4j-api</include>
         <include>org.slf4j:slf4j-log4j12</include>
+        <!-- Jersey/Jackson-based webservice -->
```

Comment:
For fun, I might try to package sparkjava in Fedora. For the purposes of this effort, I think I'm more concerned about straying too far from established standards and running into novel problems to solve which would have been easily addressed with the more well-traveled path.